### ISSUE TITLE
`make docs` in `docs/` requires docker login or missing base image

### ISSUE THREAD

**Issue by lukebond** [theupdateframework/notary#1448]
`Dockerfle` base image is `docs/base:oss` which either doesn't exist or requires docker login. Suspect Dockerfile is out of date, of docs base image depends on private hub repo.

**Comment by justincormack** [comment]
Yes the docs build is very out of dat, back from when this was a Docker project. We should rework the docs standalone and publish them on a Notary page.
